src/datasources.js

```diff
@@ -11,14 +11,85 @@ define([
     edit_source,
     select_source
 ) {
-    var vellum, dataSources;
+    var vellum, dataSources, cachedDataSources;
 
     function init(instance) {
         vellum = instance;
         dataSources = vellum.opts().core.dataSources || [];
+        cachedDataSources = {
+            fixture: {}
+        };
+    }
+
+    function cacheFixtures(data) {
+        _.map(data, function(fixture) {
```

Comment:
Should this function return a value? If not, might make more sense to use `_.each`